src/components/EventEditor.vue

```diff
@@ -9,13 +9,13 @@ div
       td {{ event.id }}
     tr
       th Start
-      td {{ event.timestamp | iso8601 }}
+      datetime(type="datetime" v-model="start")
```

Comment:
This selector is kind of strange to me, very fancy but still very inconvenient (a big calendar, touch optimized scroll wheel clock on a desktop and without the ability to select seconds).
In my opinion it's not much better than just having a string for the date and time with proper validation (but I guess proper validation is very of hard to do).

Reply:
I looked around for good datetime components, and this was the best I could find. I agree with your concerns, but I do think it is better than a plain string with validation.

I looked at how Toggl does it, and I considered splitting them into different input fields (one for date, one for time) for easier editing, but that would be a lot of extra work.

Reply:
We can make an github issue for it, it's no hurry. Should work fine for now.